src/types/IdentityDriver.ts

```diff
@@ -13,8 +13,8 @@ export abstract class IdentityDriver extends Driver implements IdentitySigner {
     return nacl.sign.detached.verify(message, signature, this.publicKey.toBytes());
   }
 
-  public equals(that: Signer | PublicKey): boolean {
-    if (!(that instanceof PublicKey)) {
+  public equals(that: { publicKey: PublicKey } | PublicKey): boolean {
```

Comment:
This doesn't really belong in a build system update PR. Also I already fixed this, see #94 , so we'll just run into unnecessary conflicts.